src/main/java/ch/njol/skript/effects/EffScriptFile.java

```diff
@@ -51,34 +52,61 @@ public class EffScriptFile extends Effect {
 
 	static {
 		Skript.registerEffect(EffScriptFile.class,
-			"(1:(enable|load)|2:reload|3:(disable|unload)) s(c|k)ript [file] %string%"
+			"(1:(enable|load)|2:reload|3:(disable|unload)) script [file|named] %string%",
+			"(1:(enable|load)|2:reload|3:(disable|unload)) skript file %string%",
```

Comment:
I think the reload and disable/unload patterns can be removed. That is, we only need to support using strings for enabling.

Reply:
I don't think there's any harm in them sticking around but if you want I can remove them :)

Reply:
Turns out we can't, since this permits directories (like `enable script file "myfolder/"`) which the scripts-only expression wouldn't.

That having been said, I suppose we could add support for fetching _multiple_ scripts from a directory path maybe? That might want to be its own effect. I will look into it :)